crates/isograph_config/src/compilation_options.rs

```diff
@@ -219,6 +227,7 @@ pub struct ConfigFileOptions {
     on_invalid_id_type: ConfigFileOptionalValidationLevel,
     no_babel_transform: bool,
     include_file_extensions_in_import_statements: bool,
+    module: ConfigFileJavascriptModule,
```

Comment:
nit: this only affects how imports of entrypoint files are generated. It doesn't affect the imports in generated files.

Maybe `entrypoint_import_type`? What do you think?

(This is not a blocker, happy to change it later)

Reply:
Yeah, let's discuss that in #328

I don't think it's a high priority, but we can tackle this if someone is blocked on the import type in generated files